lametro/templates/index/index.html

```diff
@@ -180,5 +199,10 @@ <h2 style="display: inline-block;">
 
         $('#beta-info').tooltip({html: true, trigger: 'focus click', placement: 'right'});
 
+        var tooltipTriggerList = [].slice.call(document.querySelectorAll('[data-bs-toggle="tooltip"]'))
+        var tooltipList = tooltipTriggerList.map(function (tooltipTriggerEl) {
+            return new bootstrap.Tooltip(tooltipTriggerEl)
+        })
```

Comment:
Can you say a little bit about what these lines do?

Reply:
For sure! So this is actually lifted straight from the [bootstrap 5 docs on enabling tooltips](https://getbootstrap.com/docs/5.0/components/tooltips/#example-enable-tooltips-everywhere). This was added for the tooltip above the search bar that describes suggested topics being a new feature.

I believe that this selects all the tooltips by the `data-bs-toggle` attribute and the `bootstrap.Tooltip` plugin generates the content and places it adjacent to the trigger.

Reply:
Cool, thanks!